### PR TITLE
feat: add onboarding mnemonic and congratulation views for mobile stardust

### LINKED ISSUE #4625: [Task]: Add mnemonic screens for onboarding
### Task description

Add the mnemonic screens (recovery phrase and recovery phrase validation) as router components to the mobile package and adjust the presentation.

### Requirements

- [ ] Router components added to the mobile project
- [ ] Presentation adjusted for mobile

### Creation checklist

- [ ] I have assigned this task to the correct people
- [X] I have added the most appropriate labels
- [X] I have linked the correct milestone and/or project

### LINKED ISSUE #4626: [Task]: Add success screen for onboarding
### Task description

Add success screen as router component to the mobile package and adjust the presentation.

### Requirements

- [ ] Router component added to the mobile project
- [ ] Presentation adjusted for mobile

### Creation checklist

- [ ] I have assigned this task to the correct people
- [X] I have added the most appropriate labels
- [X] I have linked the correct milestone and/or project

### PR DESCRIPTION
**Pull request by paul-boegelsack**
## Summary

This PR adds the mnemonic views as well as the congratulation screen to the mobile stardust onboarding.

## Changelog

```
- Adds profiel backup router
- Adds recovery phrase component
- Adds view mnemonic screen to onboarding
- Adds verify mnemonic screen to onbarding
```

## Relevant Issues

Closes #4625
Closes #4626

## Testing

Until the mobile bindings are ready, the `dev-mode/mobile-stardust` branch has to be utilized for testing:

1 `git checkout dev-mode/mobile-stardust` 
2 `git merge feat-mobile/new-onboarding-mnemonic-views`
3 `yarn (root directory)`
4 `yarn build (packages/desktop)`
5 `yarn start (packages/desktop)`

press `ctrl + c` to change the color theme
press `ctrl + k` to toggle the keyboard

### Platforms

> Please select any platforms where your changes have been tested.

- __Desktop__
  - [ ] MacOS
  - [ ] Linux
  - [ ] Windows
- __Mobile__
  - [ ] iOS
  - [ ] Android

### Instructions

> Please describe the specific instructions, configurations, and/or test cases necessary to __test and verify__ that your changes work as intended.

...

## Checklist

> Please tick all of the following boxes that are relevant to your changes.

- [x] I have followed the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or modified tests that prove my changes work as intended
- [ ] I have verified that new and existing unit tests pass locally with my changes
- [ ] I have verified that my latest changes pass CI workflows for testing and linting
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: packages/mobile/components/RecoveryPhrase.svelte [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+{/if}
+
+<style type="text/scss">
+    div {
+        max-width: 460px;
+        &.hide {
+            filter: blur(4px);
+        }
+        span {
+            &.errored {
```
Comment (begonaalvarezd) [inline packages/mobile/components/RecoveryPhrase.svelte:39]: In figma it shows how we should always highlight the current word you are verifying, and if wrong then it red be red
image
image

File: packages/mobile/routes/onboarding/views/profile-backup/views/VerifyMnemonicView.svelte [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+    async function completeVerification(): Promise<void> {
+        await verifyAndStoreMnemonic()
+        $profileBackupRouter.next()
+    }
+
+    function onBackClick(): void {
+        $profileBackupRouter.previous()
+    }
+
+    function onKeyPress(event: KeyboardEvent): void {
```
Comment (begonaalvarezd) [inline packages/mobile/routes/onboarding/views/profile-backup/views/VerifyMnemonicView.svelte:74]: I think we could remove this for mobile

File: packages/mobile/routes/onboarding/views/profile-backup/views/VerifyMnemonicView.svelte [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+            }
+        }
+    }
+
+    onMount(() => {
+        fillChoices()
+    })
+</script>
+
+<svelte:window on:keypress={onKeyPress} />
```
Comment (begonaalvarezd) [inline packages/mobile/routes/onboarding/views/profile-backup/views/VerifyMnemonicView.svelte:97]: same with this, we could remove it

File: packages/mobile/components/RecoveryPhrase.svelte (hunk 1 above)
Comment (begonaalvarezd) [reply packages/mobile/components/RecoveryPhrase.svelte:39]: Though we can polish this later on in a polish PR

**Review by begonaalvarezd — CHANGES_REQUESTED** [review]
Looking amazing 🔥 Great work 🚀 
Just added some comments 🙏🏼

File: packages/mobile/routes/onboarding/views/CongratulationsView.svelte [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+        <div class="relative flex flex-col items-center bg-gray-100 dark:bg-gray-900 rounded-2xl mt-10 p-10 pb-6">
+            <div class="bg-green-500 rounded-2xl absolute -top-6 w-12 h-12 flex items-center justify-center">
+                <Icon icon="success-check" classes="text-white" />
+            </div>
+            <Text type="h2" classes="mb-5 text-center">{localize('views.onboarding.congratulations.title')}</Text>
+            <Text type="p" secondary classes="mb-2 text-center"
+                >{localize('views.onboarding.congratulations.body')}</Text
+            >
+        </div>
+        {#if $isOnboardingLedgerProfile}
```
Comment (begonaalvarezd) [inline packages/mobile/routes/onboarding/views/CongratulationsView.svelte:26]: I guess we could also remove this?

**Comment by paul-boegelsack** [comment]
> Looking amazing 🔥 Great work 🚀 Just added some comments 🙏🏼

Thank you 🙏  hopefully I addressed everything! I changed the UI feedback of the verification regarding figma a bit more.

**Review by begonaalvarezd — APPROVED** [review]
🔥 neat

**Pull request merged**